### PR TITLE
UnitOfMeasure entity

### PR DESCRIPTION
**Pull request by devpuppet**
added UnitOfMeasure entity, created relationship to Ingredient entity

### CODE REVIEW

File: src/main/java/com/kkukielka/springrecipeproject/domain/Ingredient.java [hunk 1]
```diff
@@ -15,6 +15,9 @@
     @ManyToOne
     private Recipe recipe;
 
+    @OneToOne(fetch = FetchType.EAGER)
+    private UnitOfMeasure unitOfMeasure;
```
Comment (devpuppet) [inline src/main/java/com/kkukielka/springrecipeproject/domain/Ingredient.java:19]: cool name for a field!

File: src/main/java/com/kkukielka/springrecipeproject/domain/UnitOfMeasure.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class UnitOfMeasure {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
```
Comment (Qkiz288) [inline src/main/java/com/kkukielka/springrecipeproject/domain/UnitOfMeasure.java]: Why use IDENTITY strategy explicitly if it is default for @GeneratedValue ?

**Review by devpuppet — COMMENTED** [review]
I think this is sehr gut code!

File: src/main/java/com/kkukielka/springrecipeproject/domain/Ingredient.java (hunk 1 above)
Comment (devpuppet) [inline src/main/java/com/kkukielka/springrecipeproject/domain/Ingredient.java:19]: why use EAGER fetch type if it is default for this relationship?

Comment (devpuppet) [reply src/main/java/com/kkukielka/springrecipeproject/domain/Ingredient.java:19]: I wanted for other developers to know that I intentionally set this fetch type
(I am talking to myself, cool)

**Pull request merged**